pandas/io/formats/style.py

```diff
@@ -1350,8 +1352,11 @@ def pipe(self, func, *args, **kwargs):
             Function to apply to the Styler.  Alternatively, a
             ``(callable, keyword)`` tuple where ``keyword`` is a string
             indicating the keyword of ``callable`` that expects the Styler.
-        *args, **kwargs :
+        *args : iterable, optional
             Arguments passed to `func`.
+        **kwargs : dict, optional
+            A dictionary of keyword arguments passed into ``func``.
+
```

Comment:
Looks like there are too many newlines here

Reply:
Thanks @jbrockmendel,  I have removed the extra new line that was added.